.travis.yml

```diff
@@ -20,11 +20,9 @@ matrix:
     - php: 5.6
       env: SYMFONY_VERSION=2.8.*@dev
     - php: 5.6
-      env: SYMFONY_VERSION="3.0.x-dev as 2.8"
+      env: SYMFONY_VERSION="3.0.x-dev as 2.8" PHPUNIT_FLAGS="--exclude-group legacy"
```

Comment:
instead of excluding them, you should skip them in the code when they cannot run, to be able to have it working locally too (and being able to run tests for our own BC layers if 3.1 deprecates something)